src/platform/web.rs

```diff
@@ -65,8 +65,76 @@ pub trait WindowExtWebSys {
     /// For example, by default using the mouse wheel would cause the page to scroll, enabling this
     /// would prevent that.
     ///
+    /// Specifically, this will call `event.preventDefault()` for the following event types:
+    /// `touchstart`, `wheel`, `contextmenu`, `pointerdown`, `pointermove`, `keyup`, and `keydown`.
```

Comment:
This should mention that `Event.preventDefault()` is only called for `pointermove` in the case of a [chorded button interaction](https://www.w3.org/TR/pointerevents3/#chorded-button-interactions).

I know this is getting even more into the weeds, but it might be important ...

Reply:
Oh, interesting -- that's somewhat subtle. Changed!